### DISCUSSION TITLE
Are the Signage files UTF-8?

### DISCUSSION THREAD

**Discussion by FurtadoPires** [ReactiveDrop/reactivedrop_translations#1508] [Q&A] [answered]
From `misc\signage\signage_.vdf`

Can I use punctuation in those texts? Or there's a risk of glitching the text?

**Comment by Duke-M-commits** [comment, marked as answer]
As far as I understand this has been set up for a to-be-build-in-the-future pipeline which generates translated signage textures.
But since this does not exist yet you probably can do what you want.

**Reply by BenLubar** [reply]
There won't be an automated system for the signage files. When we eventually do get around to putting it in the game, someone will manually re-make all the textures using the translated text and then use the LanguagePreference proxy to load a different texture based on the player's selected language.
